sdk/core/azure-core-experimental/src/test/java/com/azure/core/experimental/http/JacksonAdapterXmlSerializableTests.java

```diff
@@ -24,7 +24,7 @@
 public class JacksonAdapterXmlSerializableTests {
     private static final SerializerAdapter ADAPTER = JacksonAdapter.createDefaultSerializerAdapter();
     private static final SimpleXmlSerializable OBJECT = new SimpleXmlSerializable(true, 10, 10.0, "10");
-    private static final String XML = "<?xml version='1.0' encoding='utf-8'?>"
+    private static final String XML = "<?xml version='1.0' encoding='UTF-8'?>"
```

Comment:
I was curious why the switch from UTF to utf? Encodings should be case insensitive.